templates/templates/blockified/cart.html

```diff
@@ -161,4 +164,4 @@
 	<!-- /wp:woocommerce/cart -->
 </div>
 <!-- /wp:group -->
-<!-- wp:template-part {"slug":"footer"} /-->
```

Comment:
By looking at other templates, we should probably keep the empty line at the bottom

Reply:
Good catch, @wavvves. I added the empty line again and also added empty lines at the Checkout templates between the checkout header and the group element so that these templates are consistent with our others.